include/zephyr/dt-bindings/clock/stm32f10x_clock.h

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Non blocking: Cf `include/zephyr/dt-bindings/clock/stm32f427_clock.h` to see how we can minimize code duplication in these files in similar situation

Reply:
> I don't understand what "Cf" means.

Sorry, "Cf" stands for `confere` as "Please see". 

> Looks odd, but maybe intentional?

That's indeed as expected (reusing existing binding when compatibles).
But I agree this is not exactly what I though as in this case as base  `stmf324_clock.h` (which contain all common pieces) is used and then complementary ` stm32f410_clock.h` or ` stm32f427_clock.h` are added on matching devices (which then also benefits to the SoCs that inlcudes those .dtsi, as stm32f429.dtsi for instance) to add the variant specific bindings.